exporters/otlp/trace/src/main/java/io/opentelemetry/exporter/otlp/trace/OtlpGrpcSpanExporter.java

```diff
@@ -35,13 +36,15 @@
 /** Exports spans using OTLP via gRPC, using OpenTelemetry's protobuf model. */
 @ThreadSafe
 public final class OtlpGrpcSpanExporter implements SpanExporter {
-
+  private static final AttributeKey<String> EXPORTER_KEY = AttributeKey.stringKey("exporter");
+  private static final AttributeKey<String> SUCCESS_KEY = AttributeKey.stringKey("success");
   private static final String EXPORTER_NAME = OtlpGrpcSpanExporter.class.getSimpleName();
-  private static final Labels EXPORTER_NAME_LABELS = Labels.of("exporter", EXPORTER_NAME);
-  private static final Labels EXPORT_SUCCESS_LABELS =
-      Labels.of("exporter", EXPORTER_NAME, "success", "true");
-  private static final Labels EXPORT_FAILURE_LABELS =
-      Labels.of("exporter", EXPORTER_NAME, "success", "false");
+  private static final Attributes EXPORTER_NAME_Attributes =
```

Comment:
should probably CAPITALIZE the ATTRIBUTES here as well. :)

Reply:
Ah, VS Code let me down here.  The "ignore capitalization" vs. "use capitalization" in find replace vs. "change symbol" hot keys always get me.  Fixed

Reply:
maybe didn't push the fix?